pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -78,7 +78,7 @@ func (ci *CloudInfo) collectInfo(ic *types.InstallConfig) error {
 func (ci *CloudInfo) getSubnet(subnetID string) (*subnets.Subnet, error) {
 	subnet, err := subnets.Get(ci.clients.networkClient, subnetID).Extract()
 	if err != nil {
-		if errors.Is(err, gophercloud.ErrResourceNotFound{}) {
+		if _, ok := err.(*gophercloud.ErrResourceNotFound); ok {
```

Comment:
this seems backwards... we should be moving towards errors.Is/As for error assertions from interface assertions

Reply:
I changed this to errors.As()